src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+html{
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```

Comment:
its not a good practice to reset styles like this. It's only allowed to use `box-sizing: border-box` for `*` selector